pkg/wrangler/context.go

```diff
@@ -187,6 +215,47 @@ func (w *Context) Start(ctx context.Context) error {
 	return nil
 }
 
+func (w *Context) WithAgent(userAgent string) *Context {
+	userAgent = fmt.Sprintf("rancher-%s-%s", settings.ServerVersion.Get(), userAgent)
```

Comment:
No, the agent is intended to be configured per handler. The controller would be less useful since in the kube-api logs we are aware of which endpoint was requested, and can infer which controller made that request. The use case here is to look at the kube-api logs and know which handlers are calling endpoints. For example, today we might see many user requests and rolebindings requests but we do not know which handler is making these requests or even if it is just one handler. This seeks to provide information on that so we can figure out who is hammering the api which should help us answer why that is happening and how to fix it if it's an issue.